### ISSUE TITLE
PHP Version

### ISSUE THREAD

**Issue by barrymieny** [Dgame/atom-php-fmt#8]
I received an error "PHP needs to be a minimum version of PHP 5.6.0", but I already have PHP 5.6.13 installed via Homebrew. Is there some place to tell php-fmt which binary to use, like in SublimeText with the "php_bin" setting?

**Comment by Dgame** [comment]
The php-executeable you set to the php config path, should have version 5.6.0. Otherwise (if php fmt did not recognize it) it's a bug.

**Comment by barrymieny** [comment]
Thanks - I completely overlooked that setting.

**Comment by Dgame** [comment]
@barrymieny So does it works?

**Comment by barrymieny** [comment]
It runs, but it doesn't do any formatting. Also, none of the options come up in the command palette like in Sublime Text.

**Comment by Dgame** [comment]
Yeah that's a limitation of Atom. Could you copy me the result you see in the developer settings? (Strg+Alt+i afaik)

**Comment by sdhou** [comment]
I've had the same problem.

**Comment by awfm9** [comment]
Same problem.
